packages/core/src/models/uischema.ts

```diff
@@ -249,3 +258,15 @@ export const isGroup = (layout: Layout): layout is GroupLayout =>
 
 export const isLayout = (uischema: UISchemaElement): uischema is Layout =>
   (uischema as Layout).elements !== undefined;
+
+export const isScopeable = (obj: object): obj is Scopable =>
+  obj !== undefined && obj.hasOwnProperty('scope');
+
+export const isScoped = (obj: object): obj is Scoped =>
+  isScopeable(obj) && obj.scope !== null;
```

Comment:
As mentioned above, the documentation says that `null` is an expected value, so it should not be excluded here.

Reply:
See above answer. This might be obj.scope !== undefined then